rfcs/0005-rfc-process.md

```diff
@@ -0,0 +1,54 @@
+---
+created-on: 2021-03-29
+freshness: 2021-03-29
+owner: @dj2
+status: draft
+---
+
+[La version française suit.](#résumé)
+
+# RFC Process
+
+## Summary
+This RFC lays out some foundations for the RFC process, mostly around
+the approval and acceptance process.
+
+## Motivation
+Without some sort of framework it is difficult to know how the RFC
+process is going to work and to align everyone in the same direction.
+There are questions around when approvals are needed, how many, how long
+do we wait. This RFC attempts to set some guard rails for this process
+and to start us in the right direction.
+
+## Proposal
+1. A proposal, from PR announcement to commit MUST be no less then 2 weeks.
+   The announcement MUST be in Slack on the appropriate channels for the
+   RFC content.
+2. An RFC MUST have approval to commit. An RFC needs approval from the
+   communities which will be affected by the RFC (or, a best guess as to
+   the impacted communities).
+   (i.e. an RFC to setup base labels needs Dev, SRE and PM approval as
+   all three communities will need to agree)
+3. If there is someone uniquely positioned to review the RFC that user
+   MUST be added for approval, this may extend the wait period outside
+   the 2 weeks if the required approver is on vacation or leave.
```

Comment:
How do we determine who is **uniquely positioned** CDS is continuing to grow and it's getting more and more difficult to understand who does what and what their specific knowledge, background, and skill is.

Reply:
I think this will have to come with time. There are a few people who we know have focused on areas that are important, but will also depend on people speaking up when the RFC is announced.